bitnami/wavefront-adapter-for-istio/templates/istio/handler.yaml

```diff
@@ -0,0 +1,93 @@
+{{- if .Values.istio.create }}
+apiVersion: {{ include "wfafi.istio.apiVersion" . }}
+kind: handler
+metadata:
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: wavefront-adapter-for-istio
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  name: {{ include "common.names.fullname" . }}-wavefront-handler
+  namespace: {{ .Values.istio.namespace }}
```

Comment:
Some templates under `templates/istio` use `.Release.Namespace` and others `.Values.istio.namespace`. Is that correct?

Reply:
Yes, some of the objects need to be deployed in the istio namespace